game/scripts/npc/items/item_trumps_fists.txt

```diff
@@ -70,7 +72,7 @@
 			"02"
 			{
 				"var_type"				"FIELD_INTEGER"
-				"corruption_armor"		"-20 -25"
+				"corruption_armor"		"20 25"
```

Comment:
Did you remove the `-` on all these values for the sake of tooltips or something? Normal Dota Desolator and Skadi seem to ignore the `-` when showing tooltips, and unless it's for tooltips I don't see any reason to make these KVs different from how normal Dota does it.